<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fft_dialog</class>
 <widget class="QDialog" name="fft_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>520</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>762</width>
    <height>520</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QFrame" name="fre_Frame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>621</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QFrame" name="Time_Frame">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>200</y>
     <width>621</width>
     <height>150</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_time">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>270</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>时域</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_fre">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>80</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>频域</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_fre_choice">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>滤波频率选择</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_size_choice">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>lg(缩放幅度)</string>
   </property>
  </widget>
  <widget class="QSlider" name="sld_fre">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>501</width>
     <height>19</height>
    </rect>
   </property>
   <property name="maximum">
    <number>1000</number>
   </property>
   <property name="value">
    <number>1000</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_fre_0hz">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>400</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0Hz</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_fre_fs">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>400</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>fs/2</string>
   </property>
  </widget>
  <widget class="QSlider" name="sld_size">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>450</y>
     <width>501</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimum">
    <number>-100</number>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_size_2_neg">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>460</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-2</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_size_2_pos">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>460</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_lowpass">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>410</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>低通滤波</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_highpass">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>410</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>高通滤波</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_size_change">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>480</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>缩放信号</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_fre_now">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>380</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>50</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_size_now">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>440</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_chan_choice">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>当前选择通道</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_chan_now">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>410</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_show">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>410</y>
     <width>91</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>信号显示</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_fs_2_pos">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>170</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>fs/2</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_fs_2_neg">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-fs/2</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_20V_neg">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>-20V</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_20V_pos">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>200</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string> 20V</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
